.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  extends: 'airbnb/base',
+  env: {
+    'browser': true,
+    'mocha': true,
+    'node': true
+  },
+  rules: {
+    'comma-dangle': ['error', 'never']
+  }
```

Comment:
Didn't we agree this will be left to airbnb default? 🤔

Reply:
We did, but I ran into problems when running `.js` `src` against `.coffee` tests. I will look into it again an see if I will be able to remedy it somehow.

Reply:
All right. I'm okay with removing the `rules` once all CoffeeScript is gone.